lib/events/athena/consumer.go

```diff
@@ -258,7 +260,7 @@ func (c *consumer) runContinuouslyOnSingleAuth(ctx context.Context, eventsProces
 			err := backend.RunWhileLocked(ctx, backend.RunWhileLockedConfig{
 				LockConfiguration: backend.LockConfiguration{
 					Backend:            c.backend,
-					LockNameComponents: []string{"athena_lock"},
+					LockNameComponents: c.consumerLockName,
```

Comment:
Do we really need lock names like `athena/foo`? Even so, we can just do something like
```go
lockName := []string{"athena", c.consumerLockName}
if c.consumerLockName == "" {
  lockName = []string{"athena_lock"}
}
...
    LockNameComponents: lockName,
```

Alternatively, just `LockNameComponents: []string{"athena_" + cmp.Or(c.consumerLockName, "lock")}` would work, if we're fine with `athena_foo` as a lock name.

Reply:
> Do we really need lock names like `athena/foo`

I like the idea of using `LockNameComponents` for constructing locks and taking care of disallowed characters.

Updated per your suggestion, thanks!